ivy.el

```diff
@@ -3956,6 +3963,24 @@ characters (previous if ARG is negative)."
   (interactive "p")
   (ivy--yank-by #'forward-char arg))
 
+(defvar ivy--pulse-overlay nil
+  "Overlay used to highlight yanked word.")
+
+(defvar ivy--pulse-timer nil
+  "Timer used to dispose of ivy--pulse-overlays.")
+
+(defun ivy--pulse-region (begin end)
+  (unless ivy--pulse-overlay
+    (setq ivy--pulse-overlay (make-overlay 0 0)))
+  (when ivy--pulse-timer
+    (cancel-timer ivy--pulse-timer))
+  (move-overlay ivy--pulse-overlay begin end (current-buffer))
```

Comment:
Nitpick: No need for `(current-buffer)`, that's already the default.

Reply:
there is a subtlety here: the default is to move the overlay inside in the buffer it is currently.

Here the overlay can change buffer, and (current-buffer) is necessary.